v2/sentryutil/sentry.go

```diff
@@ -30,6 +31,31 @@ func RecoverWithContext(ctx context.Context, transaction *sentry.Span) {
 
 // Error sends error to Sentry.
 func Error(ctx context.Context, err error) {
+	if err == nil {
+		return
+	}
+
+	_, hub := setHubToContext(ctx)
+	hub.CaptureException(err)
+	slog.Error(err.Error()) //nolint: sloglint
```

Comment:
You could call just `ErrorUnlessIgnored` with nil ignoredErrs to avoid duplication.